packager/app/retired_flags.cc

```diff
@@ -35,9 +35,6 @@ DEFINE_int32(
     num_subsegments_per_sidx,
     0,
     "This flag is deprecated. Use --generate_sidx_in_media_segments instead.");
-DEFINE_bool(generate_widevine_pssh,
```

Comment:
Please don't remove this flag. 

Add the retired flag 'generate_static_mpd' here in a similar way to 'generate_widevine_pssh' here.

Reply:
Sorry ... I mistaken ... :bow: